RELEASE.md

```diff
@@ -1,26 +1,58 @@
 
-###v0.9.6
+## Version History
 
-November, 2015 | [Release Notes](https://github.com/apache/incubator-predictionio/blob/master/RELEASE.md) have been moved to Github and you are reading them. For a detailed list of commits check [this page](https://github.com/apache/incubator-predictionio/commits/master)
+### 0.10.0
+
+Aug ??, 2016
+
+ - Make SSL optional
+ - Merge ActionML fork
+ - First Apache PredictionIO (incubating) release
+
+### 0.9.7-aml (ActionML fork)
+
+Aug 5, 2016
+
+ - changed version id so artifacts don't conflict with naming in the Salesforce sponsored project.
+ - bug fix in memory use during moving window event trim and compaction  EventStore data.
+ - update [install.sh](https://github.com/actionml/PredictionIO/blob/master/bin/install.sh) script for single line installs with options that support various combinations required by some templates.
+ 
+### v0.9.6
+
+April 11, 2015 
+
+For a detailed list of commits check [this page](https://github.com/apache/incubator-predictionio/commits/master)
 
 - Upgrade components for install/runtime to Hbase 1, Spark 1.5.2 PIO still runs on older HBase and Spark back to 1.3.1, upgrading install of Elaticsearch to 1.5.2 since pio run well on it but also runs on older versions.
 - Support for maintaining a moving window of events by discarding old events from the EventStore
 - Support for doing a deploy without creating a Spark Context
 
+### v0.9.6 (ActionML fork)
+
+March 26, 2016
+
+- Upgrade components for install/runtime to Hbase 1.X, Spark 1.5.2 PIO still runs on older HBase and Spark back to 1.3.1, upgrading install of Elasticsearch to 1.5.2 since pio run well on it but also runs on older versions.
+- Support for maintaining a moving window of events by discarding old events from the EventStore
+- Support for doing a deploy without creating a Spark Context
+
 ###v0.9.5 
 
-October 14th, 2015 | [Release Notes](https://github.com/apache/incubator-predictionio/blob/master/RELEASE.md) have been moved to Github and you are reading them. For a detailed list of commits check [this page](https://github.com/apache/incubator-predictionio/commits/v0.9.5)
+October 14th, 2015 
+
+[Release Notes](https://github.com/apache/incubator-predictionio/blob/master/RELEASE.md) have been moved to Github and you are reading them. For a detailed list of commits check [this page](https://github.com/apache/incubator-predictionio/commits/v0.9.5)
 
 - Support batches of events sent to the EventServer as json arrays
 - Support creating an Elasticsearch StorageClient created for an Elasticsearch cluster from variables in pio-env.sh
 - Fixed some errors installing PredictionIO through install.sh when SBT was not correctly downloaded
 
 ###v0.9.4
 
-July 16th, 2015 | [Release Notes](https://predictionio.atlassian.net/jira/secure/ReleaseNote.jspa?projectId=10000&version=13700)
+July 16th, 2015
+
+[Release Notes](https://predictionio.atlassian.net/jira/secure/ReleaseNote.jspa?projectId=10000&version=13700)
```

Comment:
sure, but this was in develop when I got it, all I changed was the formatting. In the name of expediency I'll leave it for another push if that's ok.